src/scenes/Koulutusmuodot/EsiJaPerusopetus/JarjestamislupaHTML/erityisetKoulutustehtavat.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from "react";
+import {
+  addIndex,
+  and,
+  compose,
+  filter,
+  find,
+  isEmpty,
+  isNil,
+  map,
+  not,
+  path,
+  propEq,
+  toUpper
+} from "ramda";
+import { useIntl } from "react-intl";
+import education from "../../../../i18n/definitions/education";
+import { getPOErityisetKoulutustehtavatFromStorage } from "helpers/poErityisetKoulutustehtavat";
+
+export default function PoOpetuksenErityisetKoulutustehtavatHtml({
+  maaraykset
+}) {
+  const intl = useIntl();
+  const locale = toUpper(intl.locale);
+  const [
+    erityisetKoulutustehtavatKoodisto,
+    setErityisetKoulutustehtavatKoodisto
+  ] = useState([]);
+
+  /** Fetch opetuksenJarjestamismuodot from storage */
+  useEffect(() => {
+    getPOErityisetKoulutustehtavatFromStorage()
+      .then(erityisetKoulutustehtavat =>
+        setErityisetKoulutustehtavatKoodisto(erityisetKoulutustehtavat)
+      )
+      .catch(err => {
+        console.error(err);
+      });
+  }, []);
+
+  const erityisetKoulutustehtavat = filter(
+    maarays =>
+      maarays.kohde.tunniste === "erityinenkoulutustehtava" &&
+      maarays.koodisto === "poerityinenkoulutustehtava",
+    maaraykset
+  );
+
+  return !isEmpty(erityisetKoulutustehtavat) &&
+    !isEmpty(erityisetKoulutustehtavatKoodisto) ? (
+    <div className={"pt-8 pb-4"}>
```

Comment:
Muissa osioissa taitaa olla mt-4 tässä ainoa luokka

Reply:
Ei näyttäis olevan ainoa.